package.json

```diff
@@ -23,12 +23,13 @@
     "test": "yarn workspaces run test",
```

Comment:
Could it make sense to run the plugin tests separately? to avoid loading too much stuff in memory

Reply:
yes but this would imply changin all the gh workflows

I can add new commands if you want to execute them separately

```json 
"test:erc20": "yarn test client-erc20"
"test:addressList": "yarn test client-addressList"
"test:multisig": "yarn test client-multisig"
```

Reply:
We can create a separate task